webapp/packages/core-app/src/NavigationTree/NavigationTreeContextMenuService.ts

```diff
@@ -35,6 +44,52 @@ export class NavigationTreeContextMenuService {
     });
   }
 
+  private async onNodeViewOptionMenuItemClick(nodeId: string, simple: boolean) {
+    const connectionId = NodeManagerUtils.connectionNodeIdToConnectionId(nodeId);
+    await this.connectionViewService.changeConnectionView(connectionId, simple);
+    await this.navNodeManagerService.refreshTree(nodeId);
+  }
```

Comment:
`changeConnectionView`

also, we probably need `try catch` construction here, to log exceptions